templates/dataEntry.html

```diff
@@ -52,6 +52,7 @@
                 </tbody>
               </table>
               <div class="focusguard focusguard-bottom" tabindex="0"></div>
+              <a type='button' id='save-button' class='btn btn-default'>Save</a>
```

Comment:
Can we style this with `width: 100%` like the `no-show-button` above? See https://github.com/cubing/ccm/blob/5393c84e9ab02f3642a4bcc82e28d7c581a572cb/css/dataEntry.import.less#L11-L12

Also, more importantly, to be compatible with how cubecomps does this, can we make this so that when the user presses enter after entering the last solve time, it focuses the "save" button? I think it's very important that you shouldn't need to use the mouse to do data entry.